Public/Src/Cache/ContentStore/Grpc/ContentStore.proto

```diff
@@ -247,18 +247,17 @@ message SelectorData {
 }
 
 message CopyFileRequest {
-    string TraceId = 1;
-    int32 HashType = 2;
-    bytes ContentHash = 3;
-    string Drive = 4;
-    int64 Offset = 5;
+    string traceId = 1;
+    int32 hashType = 2;
+    bytes contentHash = 3;
+    string drive = 4;
+    int64 offset = 5;
     CopyCompression compression = 6;
 }
 
 message CopyFileResponse {
-    ResponseHeader Header = 1;
-    bytes Content = 2;
-    int64 Index = 3;
+    bytes content = 1;
+    int64 index = 2;

```

Comment:
>index [](start = 10, length = 5)

Curious why camlCasing these but keeping the other types PascalCasing...

Reply:
proto convention is Java: members smallCamel and types BigCamel. The code generator uses .NET convention when it generates C#.

Reply:
I'm curious, should we switch to the new convention or should we keep using the old "unconventional" style?

---
In reply to: [272773030](https://github.com/Microsoft/BuildXL/pull/98#discussion_r272773030) [](ancestors = 272773030)